commands/init.go

```diff
@@ -180,18 +191,69 @@ func (a *initFnCmd) init(c *cli.Context) error {
 		return err
 	}
 
-	// TODO: why don't we treat "docker" runtime as just another language helper? Then can get rid of several Docker
-	// specific if/else's like this one.
-	if runtimeSpecified && runtime != common.FuncfileDockerRuntime {
-		err := a.generateBoilerplate(dir, runtime)
+
+	if (initImage != ""){
+
+		fmt.Println("Building from init-image: " + initImage)
+
+		// Run the initImage
+		var c1ErrB, c2ErrB bytes.Buffer
+
+		c1 := exec.Command("docker", "run", "-e", "FN_FUNCTION_NAME="+a.ff.Name, initImage)
+		c1.Stderr = &c1ErrB
+
+		c2 := exec.Command("tar", "-x")
+		c2.Stderr = &c2ErrB
+		c2.Stdin, _ = c1.StdoutPipe()
+		c2.Stdout = os.Stdout
+
+		_ = c2.Start()
+		c1_err := c1.Run()
+		c2_err := c2.Wait()
```

Comment:
normally I just use https://golang.org/pkg/os/exec/#Cmd.CombinedOutput cuz I'm super lazy, may clean up error handling/etc. here if it's possible to use it (didn't investigate further sorry)

Reply:
For c1 (ie `docker run`) we need to handle them separately - stdout goes to c2.stdIn & stderr goes to the user.

For c2 (ie `tar -x`) we expect stdout to be empty.  Any errors should go to the user.